packages/DatePicker/DatePicker.md

```diff
@@ -0,0 +1,41 @@
+### Usage Criteria
+
+#### Overlay Date Picker
+
+- Use the overlay `DatePicker` whenever possible as it is the most accessible solution; the input form field is type-accessible and optimized for mobile customers
+- The overlay `DatePicker` component includes an input form field to allow manual key-in of dates
+- The overlay `DatePicker` opens when the input form field receives focus
+
+```jsx
+initialState = { date: moment() }
```

Comment:
If moment is recommended, should we document what it is for those who aren't aware?